live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -69,8 +69,24 @@
 # Also, we can't use the "service" module, as that fails when running
 # inside the chroot environment, so we have to use "command" here.
 #
+# Additionally, we raise the log level to help diagnose problems; in the
+# event that networking fails, this additional information is valuable.
+#
+#
```

Comment:
one too many empty comment lines

Reply:
shoot. thanks for catching that. I wish we had a style checker for these ansible files.